src/main/golo/async.golo

```diff
@@ -85,6 +85,17 @@ augment gololang.concurrent.async.Promise {
     }): start()
     return this: future()
   }
+
+  ----
+  You can define a promise which runs "stuff" inside a Worker
+  You have to pass a `WorkerEnvironment` to `initializeWithinWorker` method
+  ----
+  function initializeWithinWorker = |this, env, closure| {
+    env: spawn(|message| {
+      this: initialize(closure)
+    }): send("")
```

Comment:
What can I send?
null generate an error
and I can't find a use case where the message is useful

On Sun Jan 18 2015 at 11:12:26 PM Julien Ponge <notifications@github.com>
wrote:

> In src/main/golo/async.golo
> <https://github.com/golo-lang/golo-lang/pull/239#discussion-diff-23137585>
> :
>
> > @@ -85,6 +85,17 @@ augment gololang.concurrent.async.
>
> Promise {
> >      }): start()
> >      return this: future()
> >    }
> > +
> > +  ----
> > +  You can define a promise which runs "stuff" inside a Worker
> > +  You have to pass a `WorkerEnvironment` to `initializeWithinWorker` method
> > +  ----
> > +  function initializeWithinWorker = |this, env, closure| {
> > +    env: spawn(|message| {
> > +      this: initialize(closure)
> > +    }): send("")
>
> Why send an empty string?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/golo-lang/golo-lang/pull/239/files#r23137585>.
>